tests/functional/syntax/test_block.py

```diff
@@ -153,3 +153,31 @@ def foo() -> uint256:
 @pytest.mark.parametrize("good_code", valid_list)
 def test_block_success(good_code):
     assert compiler.compile_code(good_code) is not None
+
+
+valid_list = [
+    """
+@external
+def foo() -> uint256:
+    return block.blobbasefee
+    """,
+    """
+@external
+def foo() -> uint256:
+    a: uint256 = 5
+    a = block.blobbasefee
+    return a
+    """,
+    """
+@external
+def foo() -> uint256:
+    a: uint256 = block.blobbasefee
+    return a
+    """,
+]
+
+
+@pytest.mark.requires_evm_version("cancun")
+@pytest.mark.parametrize("good_code", valid_list)
+def test_block_blob_success(good_code):
+    assert compiler.compile_code(good_code) is not None
```

Comment:
do we need to actually test the behavior? or at least check that blobbasefee is in the assembly